admin/pg12/upgrade-to-pg12

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -e
+
+docker-compose down
```

Comment:
It should source `admin/lib/common.lib.bash` and replace `docker`/`docker-compose` commands:
```suggestion
$DOCKER_COMPOSE_CMD down
```

Reply:
Fixed in https://github.com/metabrainz/musicbrainz-docker/pull/146/commits/88b0c7110f69f8f7ba5bcd838491ca749b4daf38